analytics/population/nc_updated_pop_comparison.ipynb

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    pop_compare['pop_diff(%)'] = (abs(pop_compare['total_population']-pop_compare['pop_arcGIS'])/((pop_compare['total_population']+pop_compare['pop_arcGIS'])/2))*100
</pre><p>Please remove symbols from names. We can just call it pop_diff_percent</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Made the changes. </p>